libraries/protocol/include/graphene/protocol/types.hpp

```diff
@@ -239,7 +239,9 @@ GRAPHENE_DEFINE_IDS(protocol, protocol_ids, /*protocol objects are not prefixed*
                     (vesting_balance)
                     (worker)
                     (balance)
-                    (htlc))
+                    (htlc)
+                    (voting_statistics)
```

Comment:
Move into plugin.

Reply:
When I move this into the plugin, `graphene_chain` will depend on the plugin, since the `vot*_objs` are there. Is this okay? Or should I just move the whole objects inside the plugin dir?